emba

```diff
@@ -368,6 +368,20 @@ main() {
   emba_parameter_parsing "$@"
   set_log_paths
 
+  # Validate RESCAN_SBOM mode requirements
+  if [[ ${RESCAN_SBOM} -eq 1 ]]; then
```

Comment:
We have started quoting of all variable usages a while ago. This is not needed in every single case but it makes further steps simpler as we do not need to check for issues around this.
```
  if [[ "${RESCAN_SBOM}" -eq 1 ]]; then
```

Thanks